.gitignore

```diff
@@ -128,6 +128,13 @@ infrastructure/.terraform.lock.hcl
 .vscode
 *.code-workspace
 
+# SSH keys
+*.pem
+*.pub
+
+# 1Password integration
+.op/
+
```

Comment:
Can you tell me what step created this folder?

Reply:
When we configured the `op` plugin, it created a `.op/` directory, which contains a `plugins/` directory, which in turn includes a `ccdl.json` file. This json file includes my op `account_id`, the plugin `entrypoint`, and a field called `credentials` which is an object that includes the type of plugin, the credential_type (which is an access_key), my vault_id, the item_id, etc.

Reply:
Ok so I think that makes sense to keep we might want to add this to documentation for a .gitignore defaults file.